include/user.php

```diff
@@ -0,0 +1,128 @@
+<?php 
+
+function getUser($accountId){
+    $user = dbQuery('
+    SELECT *
+    FROM trackerSignUp
+    WHERE accountId = :accountId', 
+    [
+        'accountId' => $accountId, 
+        
+    ])->fetch();                             
+return @$user; //this is an array
+}
+
+
+
+function attemptLogin($email, $password){ 
+    $result = dbQuery('
+        SELECT accountId
+        FROM trackerSignUp
+        WHERE email = :email AND password = :password', //just a string parameter
+        [
+            'email' => $email,
+            'password' => $password
+            
+        ])->fetch();                            //where email/password matches -case insensitive email
+    return $result['accountId'];
+}    
+
+
+
+function insertNewProfileEdit($email, $age, $gender, $occupation, $occupationStressLevel, $typeBeverage, $beverageOther, $lifeEvent){
+dbQuery(
+        '   
+            UPDATE trackerSignUp
+            SET  age = :age,
+            gender = :gender, 
+            occupation = :occupation, 
+            occupationStressLevel = :occupationStressLevel, 
+            typeBeverage = :typeBeverage, 
+            beverageOther = :beverageOther,
+            lifeEvent = :lifeEvent
+            WHERE email = :email
+        ',
+        [   'age' => $age,
+            'gender' => $gender, 
+            'occupation' => $occupation, 
+            'occupationStressLevel' => $occupationStressLevel, 
+            'typeBeverage' => $typeBeverage, 
+            'beverageOther' => $beverageOther,
+            'email' => $email,
+            'lifeEvent' => $lifeEvent
+
+        ]
+       
+        );
+}
+
+function insertNewHappinessQuestionnaireInput($email, $happinessQuestion1, $happinessQuestion2, $happinessQuestion3, $happinessQuestion4){
```

Comment:
I'm a bit confused by this. Why are you storing these as columns in the user table rather than separate rows in a new table (like the questionnaire table)